video_encoding/files.py

```diff
@@ -39,8 +41,15 @@ def _get_video_info(self):
         if not hasattr(self, '_info_cache'):
             encoding_backend = get_backend()
             try:
-                path = os.path.abspath(self.path)
-            except AttributeError:
-                path = os.path.abspath(self.name)
-            self._info_cache = encoding_backend.get_media_info(path)
+                _path = getattr(self, 'path', self.name)
+            except NotImplementedError:
+                _path = self.name
+            try:
+                path = default_storage.path(_path)
+            except NotImplementedError:
+                path = default_storage.url(_path)
+            try:
+                self._info_cache = encoding_backend.get_media_info(path)
+            except FFmpegError:
```

Comment:
Why did you decide to swallow this exception? I would argue if something goes wrong, we should have to explicitly deal with it in the application code. Furthermore, I guess if we can't get the media information there might be something wrong with the file itself and we cannot perform any other operation on it either way.